GithubHandler.py

```diff
@@ -25,6 +25,21 @@ def __init__(
         self.prs = self.repo.get_pulls(state="open", sort="created", base=main_branch)
         self.prs_nums = [pr.number for pr in self.prs]
         self.prs_dict = {k: v for k, v in zip(self.prs_nums, self.prs)}
+        self.accepted_file_extensions = [
```

Comment:
Lists longer than a grocery run before a snowstorm should be split up. Your extensions list could use a little whitespace breath. Put each extension on its own line, make it clear and easy. No need to squish them together like sardines.